tests/interfaces/protocols-configure.spec.ts

```diff
@@ -314,6 +314,34 @@ describe('ProtocolsConfigure', () => {
           .to.be.rejectedWith(DwnErrorCode.ProtocolsConfigureInvalidActionMissingOf);
       });
 
+      it('rejects protocol definitions with `can: query` in non-role rules', async () => {
```

Comment:
Sanity: is this just a temporary measure, or we actually think that query only makes sense for protocol actors defined by roles?

Reply:
This is a temporary measure

Reply:
Got it!